ydb/core/driver_lib/cli_utils/cli_cmds_tenant.cpp

```diff
@@ -606,6 +620,40 @@ class TClientCommandTenantAddPools
     }
 };
 
+class TClientCommandTenantChangeQuotas
+    : public TTenantClientGRpcCommand<Ydb::Cms::V1::CmsService,
+                                      Ydb::Cms::AlterDatabaseRequest,
+                                      Ydb::Cms::AlterDatabaseResponse,
+                                      decltype(&Ydb::Cms::V1::CmsService::Stub::AsyncAlterDatabase),
+                                      &Ydb::Cms::V1::CmsService::Stub::AsyncAlterDatabase> {
+
+    uint64_t DataSizeHardQuota;
+    uint64_t DataSizeSoftQuota;
+public:
+    TClientCommandTenantChangeQuotas()
+        : TTenantClientGRpcCommand("change", {}, "Change data size quotas for database")
+    {}
+
+    void Config(TConfig& config) override {
+        TTenantClientGRpcCommand::Config(config);
+        config.SetFreeArgsMin(0);
+        config.Opts->AddLongOption("data-size-hard-quota", "A maximum data size in bytes, new data will be rejected when exceeded")
+            .RequiredArgument("NUM").StoreResult(&DataSizeHardQuota);
+        config.Opts->AddLongOption("data-size-soft-quota", "Data size in bytes (lower than data_size_hard_quota), at this value new data ingestion is re-enabled again")
+            .RequiredArgument("NUM").StoreResult(&DataSizeSoftQuota);
+    }
+
+    void Parse(TConfig& config) override
+    {
+        TTenantClientGRpcCommand::Parse(config);
+
+        GRpcRequest.mutable_database_quotas()->set_data_size_hard_quota(DataSizeHardQuota);
```

Comment:
data-size-hard-quota option may be not specified, better to check it in ParseResult, like below